### DISCUSSION TITLE
Bootstrap 5 Nuxt module

### DISCUSSION THREAD

**Discussion by KoujiSano** [twbs/bootstrap#40587] [Show & Tell]
We have developed a Nuxt module for integrating Bootstrap 5 with Nuxt.

https://usebootstrap.org/

https://nuxt.com/modules/use-bootstrap

**Comment by julien-deramond** [comment]
Good job folks!
At some point, if you get some adoption, we could study with the core team if your Nuxt module could be referenced around here: https://getbootstrap.com/docs/5.3/getting-started/javascript/#usage-with-javascript-frameworks.
Please keep us in touch 🤗

**Reply by KoujiSano** [reply]
Thank you! 
We would be delighted if our Nuxt module could be referenced in the official Bootstrap documentation. 
Please provide instructions if there’s anything we should do. 
If there’s anything else you need from us, please feel free to reach out.

**Reply by julien-deramond** [reply]
Currently, we aim to provide a list of references for Bootstrap users who want to integrate Bootstrap with the main three frameworks: Angular, React, and Vue. For Vue, we already have [BootstrapVue](https://bootstrap-vue.org/) for Bootstrap 4 and [BootstrapVueNext](https://bootstrap-vue-next.github.io/bootstrap-vue-next/) for Vue 3, which is currently in alpha.

While I'm not fully familiar with the entire Vue ecosystem, we are always open to new and valuable contributions. If your Nuxt module gains traction and becomes a go-to reference for Vue users, we would certainly consider adding it to our list. At this moment, feel free to ping me around here :)

**Reply by KoujiSano** [reply]
Thank you for your encouraging message! We are thrilled about the possibility of our Nuxt module being referenced in the Bootstrap documentation. We will keep you updated on our progress and any adoption we achieve. Please feel free to reach out if you have any further suggestions or questions.

**Comment by KoujiSano** [comment]
_usebootstrap_ has undergone a significant update.

### ColorManagement
[Color Management](https://usebootstrap.org/lang-en/getting-started/color-management)


For example, the Accordion now supports AllColor. 

Accordion Similarly, the Dropdown also supports AllColor. Dropdown

[Accordion](https://usebootstrap.org/lang-en/components/accordion#themeColors)

Similarly, the Dropdown also supports AllColor. Dropdown

[Dropdown](https://usebootstrap.org/lang-en/components/dropdowns#extend)

### Extensive Extend Components

[Extend](https://usebootstrap.org/lang-en/extend)

### Partial HTML String Generation

[StaticHTMLFragmentRendering](https://usebootstrap.org/lang-en/extend/server/static)

### Server and Client Data Integration Without Scripting

[Integration](https://usebootstrap.org/lang-en/integration)

**Comment by dagnelies** [comment]
I quickly experimented with it. The first impression is quite good! 👍
